### ISSUE TITLE
snippets fail to compile on ubuntu 22.04 because libgnomeui-2 and libgnomevfs-2 no longer provided by Canonical

### ISSUE THREAD

**Issue by pflagerd** [d-widget-toolkit/dwt#112]
https://askubuntu.com/questions/1232446/libgnome-2-0-missing-from-20-04

https://wiki.gnome.org/Attic/LibgnomeMustDie

https://www.unix.com/man-page/opensolaris/3/libgnomevfs-2/

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=850268

**Comment by jacob-carlborg** [comment]
This has recently been fixed: https://github.com/d-widget-toolkit/dwt/pull/109.

**Comment by pflagerd** [comment]
```
neon-dwt:~/Desktop/dwt> cat /etc/os-release 
PRETTY_NAME="KDE neon 5.26"
NAME="KDE neon"
VERSION_ID="22.04"
VERSION="5.26"
VERSION_CODENAME=jammy
ID=neon
ID_LIKE="ubuntu debian"
HOME_URL="https://neon.kde.org/"
SUPPORT_URL="https://neon.kde.org/"
BUG_REPORT_URL="https://bugs.kde.org/"
PRIVACY_POLICY_URL="https://kde.org/privacypolicy/"
UBUNTU_CODENAME=jammy
```

```
neon-dwt:~/Desktop/dwt> dmd --version
DMD64 D Compiler v2.100.0
Copyright (C) 1999-2022 by The D Language Foundation, All Rights Reserved written by Walter Bright
```

```
neon-dwt:~/Desktop/dwt> dub --version
DUB version 1.29.0, built on May 13 2022
```

Following the instructions in README.markdown, I created main.d.

```neon-dwt:~/Desktop/dwt> dub --single main.d 
parsePackageRecipe dub.sdl
Performing "debug" build using /home/oy753c/.bin/dmd2/linux/bin64/dmd for x86_64.
dwt:base 1.0.5+swt-3.4.1.commit.130.gfe047f3: building configuration "library"...
base/src/java/nonstandard/SafeUtf.d-mixin-12(13,1): Error: undefined escape hex sequence \Ut
base/src/java/nonstandard/UnsafeUtf.d-mixin-12(13,1): Error: undefined escape hex sequence \Ut
/home/oy753c/.bin/dmd2/linux/bin64/dmd failed with exit code 1.
```

**Comment by pflagerd** [comment]
Seems the offending line in dwt/base/src/java/nonstandard/UtfBase.d is this one:

```
# line 11 "java\nonstandard\UtfBase.d"
```

which may be resolved on neon thus:

```
# line 11 "java\nonstandard\UtfBase.d"
```

**Comment by jacob-carlborg** [comment]
Is this issue solved now when https://github.com/d-widget-toolkit/dwt/pull/113 is merged?

**Comment by pflagerd** [comment]
Yes. Tested on Ubuntu 22.04.  As at afcfe87cefa6800b429fabb8007997449c38626c  on your 4.7   dub --single tools/build_snippets.d builds without failure.

**Comment by jacob-carlborg** [comment]
Fixed in #113 and #109.
